Source/WTF/wtf/text/SuperFastHash.h

```diff
@@ -156,6 +163,12 @@ class SuperFastHash {
         return StringHasher::finalize(computeHashImpl<T, Converter>(data, length));
     }
 
+    template<typename T, typename Converter = DefaultConverter>
+    static constexpr unsigned computeHash(std::span<const T> data)
+    {
+        return StringHasher::finalize(computeHashImpl<T, Converter>(data.data(), data.size()));
```

Comment:
Obviously it's time for `computeHashImpl` to learn about `span`! :-)

Reply:
I agree. I'll follow-up though as this patch is already large. There is also a very slight performance concern since this is likely hot and std::span does bounds checking.